GUI/window.py

```diff
@@ -0,0 +1,89 @@
+
+from kivy.app import App
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.recycleview.views import RecycleDataViewBehavior
+from kivy.uix.button import Button
+from kivy.properties import BooleanProperty, ListProperty, StringProperty, ObjectProperty
+from kivy.uix.recyclegridlayout import RecycleGridLayout
+from kivy.uix.behaviors import FocusBehavior
+from kivy.uix.recycleview.layout import LayoutSelectionBehavior
+from kivy.uix.popup import Popup
+from kivy.config import Config
+Config.set('graphics', 'width', '1024')
+Config.set('graphics', 'height', '650')
+Config.set('graphics', 'resizable', False)
+
+# Window tables with editable data rows
+#Uses test.kv as a config file

```

Comment:
test.kv is only for testing purposes or is it the actual config file? 

If it is the config file then it should be renamed as such

Reply:
It is not a permanent config file, just for the display of the current tables/columns. However, it won't be necessarily needed later on. It is an example/placeholder of how to configure the window.